install_cli.php

```diff
@@ -270,6 +275,9 @@
 	}
 	$modules['Teams'] = $opts['u']*($modules['Teams']/$modules['Users']);
 	$modules['Users'] = $opts['u'];
+	if (isset($opts['tba'])) {
+		$modules['ACLRoles'] = $opts['u'] / 2;

```

Comment:
This proportion doesn't seem realistic to me. I would better divide by 10, or even 20.